v3/src/models/formula/functions/function-utils.ts

```diff
@@ -33,3 +34,23 @@ equal = (a: any, b: any): boolean => {
 export const evaluateNode = (node: MathNode, scope?: FormulaMathJsScope) => {
   return node.compile().evaluate(scope)
 }
+
+// This function is used to get the root scope (an instance of our FormulaMathJSScope) within custom MathJS functions.
+// When the formula expression is executed, the initially passed scope can be wrapped in MathJS's PartitionedMap, which
+// is used to store temporary values. This function retrieves the root scope from the PartitionedMap.
+// This approach has been recommended by a MathJS maintainer. For more details, see:
```

Comment:
I think it's safe to call him _the_ MathJS maintainer. 😉 
```suggestion
// This approach has been recommended by the MathJS maintainer. For more details, see:
```